### DISCUSSION TITLE
How check not clickable element

### DISCUSSION THREAD

**Discussion by Oleksii-QA** [webdriverio/webdriverio#9725] [Q&A] [answered]
Hello. Need help for cheking not clickable element. 

There is expect for clickable element
```
const elem = await $('#elem')
await expect(elem).toBeClickable()
```

My element is not clickable and should not be clickable. How can I check this?

**Comment by Oleksii-QA** [comment, marked as answer]
Looks like I can use `not.toBeClickable()`?

**Reply by erwinheitzman** [reply]
This is indeed correct as it will invert/flip the assertion

**Reply by Oleksii-QA** [reply]
Thanks a lot
